### DISCUSSION TITLE
Is it possible to select multiple polygons and rotate them together?

### DISCUSSION THREAD

**Discussion by MagnusPladsen** [geoman-io/leaflet-geoman#1375] [Questions & Help] [answered]
Is it possible to select multiple polygons and rotate them together? I have a function that creates polygons for a field on a map. I have testet with layerGroup and featureGroup but i cant seem to make this work.

For example shift+drag to select multiple layers/polygons then rotate or move them.
I have been stuck at this for many days now without any really progress.

<img width="1123" alt="image" src="https://github.com/geoman-io/leaflet-geoman/assets/93226629/3a0c3ae6-1fef-4c68-b8a1-beec5f759e7d">

**Comment by Falke-Design** [comment, marked as answer]
No we don't have a built in feature for that

**Reply by MagnusPladsen** [reply]
Do you have any idea of how i can do this?
